vendor/k8s.io/kube-scheduler

```diff
@@ -0,0 +1 @@
+../../staging/src/k8s.io/kube-scheduler/
```

Comment:
do we have a list of what has to happen (and who can/will do it) for new staging dirs?

* publishing bot setup
* meta info at https://k8s.io/apiserver?go-get=1, etc
* ...

Reply:
@liggitt [the publishing bot](https://github.com/kubernetes/publishing-bot) will help us do such new staging dirs.